libs/src/oracle-sdk-v2/services/oracles/factory.ts

```diff
@@ -11,14 +11,12 @@ export type GqlConfig = {
   chainId: ChainId;
   type: OracleType;
   address: string;
-  addErrorMessage: (message: ErrorMessage) => void;
 };
 export type Config = GqlConfig[];
 
-export const Factory =
-  (config: Config): ServiceFactory =>
-  (handlers: Handlers): Service => {
-    const services: Service[] = config.map((config) => {
+export const Factory = (config: Config): ServiceFactory[] => {
```

Comment:
this factory used to return a single "service" from multiple configurations. it now returns an array of services. this was required so we can know which configuration is erroring in the oracle context